src/compiler/crystal/command.cr

```diff
@@ -102,7 +102,11 @@ class Crystal::Command
     when File.file?(command)
       run_command(single_file: true)
     else
-      error "unknown command: #{command}"
+      if command =~ /.+#{File::SEPARATOR}.+/ || command.ends_with?(".cr")
+        error "file not found: #{command}"
```

Comment:
Please use the same error message you get when you do `crystal run nonexistent.cr`:

```crystal
error "File #{command} does not exist"
```

Reply:
I'd also suggest adding (single-)quotes around the filename